### PR TITLE
Bugfix: no scrollfix when no edit switch fixes #276.

### LINKED ISSUE #276: Scroll spy will break the css on the assignment grading page
The grading page for a Moodle assignment doesn't have an edit toggle (M4+). The editToggle.addEventListener will throw an error if editToggle is null.
Simple detection will fix this.
```
let editToggle = document.querySelector('form.editmode-switch-form');
if (!editToggle) {
    return;
}
editToggle.addEventListener('click', () => {
...
```

### PR DESCRIPTION
**Pull request by lucaboesch**
Thanks, @sharpchi!

### CODE REVIEW

File: amd/src/scrollspy.js
Comment (t-schroeder) [inline amd/src/scrollspy.js:41]: Technically using the wrong indentation level

**Review by t-schroeder — COMMENTED** [review]
Seems good.

**Comment by lucaboesch** [comment]
@t-schroeder I saw now with a day's difference what you meant by indentation and fixed it. It would be lovely if you could go review again and then make use of the "Approve" button if you like what you see.

Best,
Luca

**Review by t-schroeder — APPROVED** [review]
Thanks for fixing the indentation. I know, it's a bit nit-picky. :)

**Comment by t-schroeder** [comment]
I approved this but it still shows "Review required". What needs to happen still?

**Review by t-schroeder — APPROVED** [review]
Approved

Edit: I approved these changes again, this time ticking the "Viewed" checkbox for each changed file. But GitHub still doesn't like it.

**Comment by t-schroeder** [comment]
It says "At least 1 approving review is required by reviewers with write access." Is the problem that I don't have write access? @lucaboesch do you know?

**Comment by lucaboesch** [comment]
Yeah, probably…
I'll have to ping @abias if he can push this through.

**Comment by abias** [comment]
I have to admit that I do not understand why @lucaboesch is not able to merge this PR has he has write permissions to this repo and as there is a approval by @t-schroeder . Maybe it's a problem that @t-schroeder does not have write permissions to the repo himself and thus the approval may not count.

Anyway, I will merge this now. Sorry for the delay.

**Pull request merged**
